utils/prometheus/Makefile

```diff
@@ -0,0 +1,62 @@
+#
+# Copyright (C) 2018 Jeffery To
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=prometheus
+PKG_VERSION:=2.2.1
+PKG_RELEASE:=1
+
+PKG_SOURCE_PROTO:=git
+PKG_SOURCE_URL:=https://github.com/prometheus/prometheus.git
+PKG_SOURCE_VERSION:=f8dcf9b272c9ab1a6148ccc22c4de6d4403c601d
+
+PKG_LICENSE:=Apache-2.0
+PKG_LICENSE_FILES:=LICENSE
+PKG_MAINTAINER:=Paul Spooren <spooren@informatik.uni-leipzig.de>
+
+PKG_BUILD_DEPENDS:=golang/host
+PKG_BUILD_PARALLEL:=1
+PKG_USE_MIPS16:=0
+
+GO_PKG:=github.com/prometheus/prometheus/
+GO_PKG_BUILD_PKG:=github.com/prometheus/prometheus/cmd/prometheus/
```

Comment:
This only builds `prometheus` and not `promtool`, is that the intended behaviour? Would `promtool` be useful on the target device? (FYI I've never used Prometheus so I don't know what `promtool` does.)

Reply:
`promtool` is quite big and so I though of rather validation the configurations locally and only install the basic `prometheus` daemon on the node.